v3/plugins/bitlang/cobol/5.0.6/meta.yaml

```diff
@@ -6,7 +6,7 @@ type: VS Code extension
 displayName: COBOL
 title: COBOL
 description: COBOL, JCL, PL/I & DIR syntax support
-icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+icon: https://raw.githubusercontent.com/spgennard/vscode_cobol/v5.2.2/images/cobol.png
```

Comment:
should we move icon to the registry ? to make offline registry working ?

Reply:
or it's part of "offline" to download icon and store it in the registry

Reply:
I'm not sure we are able do include all the images to the plugin registry repository because of licenses.
To have the offline version, all the images could be downloaded and included to the plugin registry image by `build.sh` script.

Reply:
Even the current registry contains external URis. So, it's not a problem to use proper images.